kokoro/presubmit/presubmit.sh

```diff
@@ -51,7 +51,7 @@ function check() {
 }
 
 function run_clang_format() {
-  find . -name "*.h" -o -name "*.cpp" -o -name "*.mm" -o -name "*.proto" | xargs $CLANG_FORMAT -i -style=Google
+  find . -path ./core/vulkan/cc/include/vulkan -prune -o \( -name "*.h" -o -name "*.cpp" -o -name "*.mm" -o -name "*.proto" \) -print| xargs $CLANG_FORMAT -i -style=Google
```

Comment:
Add a small comment above, e.g.:
`# Do not format files under core/vulkan/cc/include/vulkan/`